tests/integration/compile/compile-es6-spec.js

```diff
@@ -0,0 +1,355 @@
+var compileES6 = require('../../../src/compile/compile-es6');
+var _ = require('underscore');
+var path = require('path');
+
+describe('compile ES6', function() {

```

Comment:
Added a suite of tests that flex our ES6 compiler directly, making sure all the features we care about get transpiled.